frontend/src/lib/components/core/Block/Block.tsx

```diff
@@ -55,7 +55,11 @@ const BlockNodeRenderer = (props: BlockPropsWithWidth): ReactElement => {
   const { node } = props
 
   // Allow columns to create the specified space regardless of empty state
```

Comment:
Nit: Could you just update this comment to include chat messages?

Reply:
Updated 👍